lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexLauncher.java

```diff
@@ -39,22 +39,10 @@ private VortexLauncher() {
   /**
    * Launch a Vortex job using the local runtime.
    */
-  public static LauncherStatus launchLocal(final String jobName,
-                                           final Class<? extends VortexStart> vortexUserCode,
-                                           final int numOfWorkers,
-                                           final int workerMemory,
-                                           final int workerCores,
-                                           final int workerCapacity) {
+  public static LauncherStatus launchLocal(final Configuration vortexConf) {
```

Comment:
Could you add a ``{@link}`` here to ``VortexConfHelper`` such that the user knows how to generate the ``Configuration``?